### PR TITLE
PROPOSE: DIDAuthZ

### PR DESCRIPTION
**Pull request by llorllale**
Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

File: features/0309-didauthz/README.md [hunk 1]
```diff
@@ -0,0 +1,309 @@
+# Aries RFC 0309: DIDAuthZ
```
Comment (swcurran) [inline features/0309-didauthz/README.md]: Why this title?  It's using DIDComm, but the authorization is coming from a verifiable credential. It seems like an odd name for this.  Again, not a big deal - I just don't get it.

File: features/0309-didauthz/README.md [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+## Motivation
+
+In the online world, individuals frequently consent to a service provider
+gaining access to their protected resources located on a different service
+provider. Individuals are challenged with an authentication mechanism,
+informed of the resources being requested, consent to the use of their
+resources, and can later revoke access at any time. OAuth 2.0<sup>[6]</sup>,
+and other frameworks built on top of it, were developed to address this need.
+
+A DIDComm protocol<sup>[2]</sup> can address these use cases and enhance them
```
[the remark this answers is not in the document]
Comment (llorllale) [reply features/0309-didauthz/README.md]: I originally had the superscripts linked to `<span>` tags in the footnotes, but `check_links.py` complained about it. Would be nice to have, agreed.

File: features/0309-didauthz/README.md (hunk 1 above)
Comment (llorllale) [reply features/0309-didauthz/README.md]: Noted.

**Comment by llorllale** [comment]
@swcurran @dhh1128 @kdenhartog can we merge this?

**Pull request merged**